<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyPairWidget</class>
 <widget class="QWidget" name="KeyPairWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="basicKeyFeatures">
     <property name="title">
      <string>Basic Key Features</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="basicKeyFeaturesLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="keySizeLabel">
        <property name="text">
         <string>Key Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="keySizeComboBox">
        <item>
         <property name="text">
          <string>2048</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4096</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="algorithmLabel">
        <property name="text">
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="algorithmComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="keyWrappingGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Key Wrapping</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="keyWrappingGroupLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="keyWrapping">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="keyWrappingLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="wrappingAlgorithmLabel">
           <property name="text">
            <string>Wrapping Algorithm</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="wrappingAlgorithmComboBox">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="gui::widget::PasswordWidget" name="keyPassword" native="true">
        <property name="repeat" stdset="0">
         <bool>true</bool>
        </property>
        <property name="name" stdset="0">
         <string>Wrapping Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::widget::PasswordWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/gui/widget/PasswordWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
